docs/src/database_import.md

```diff
@@ -212,31 +215,31 @@ spec:
         key: password
 ```
 
-There are a few things you need to be aware of when using the `monolith` type:
+Be aware of the following when using the `monolith` type:
 
 - It requires an `externalCluster` that points to an existing PostgreSQL
-  instance containing the data to import (for more information, please refer to
-  ["The `externalClusters` section"](bootstrap.md#the-externalclusters-section))
+  instance containing the data to import. (For more information, see
+  [`externalClusters`](bootstrap.md#the-externalclusters-section).)
 - Traffic must be allowed between the Kubernetes cluster and the
-  `externalCluster` during the operation
+  `externalCluster` during the operation.
 - Connection to the source database must be granted with the specified user
-  that needs to run `pg_dump` and retrieve roles information (*superuser* is
-  OK)
-- Currently, the `pg_dump -Fc` result is stored temporarily inside the `dumps`
+  that needs to run pg_dump and retrieve roles information (superuser is
+  OK).
+- Currently, the `pg_dump -Fc` result is stored temporarily in the `dumps`
   folder in the `PGDATA` volume, so there should be enough available space to
-  temporarily contain the dump result on the assigned node, as well as the
-  restored data and indexes. Once the import operation is completed, this
-  folder is automatically deleted by the operator.
-- At least one database to be specified in the `initdb.import.databases` array
-- Any role that is required by the imported databases must be specified inside
-  `initdb.import.roles`, with the limitations below:
-    - The following roles, if present, are not imported:
-      `postgres`, `streaming_replica`, `cnp_pooler_pgbouncer`
-    - The `SUPERUSER` option is removed from any imported role
-- Wildcard `"*"` can be used as the only element in the `databases` and/or
-  `roles` arrays to import every object of the kind; When matching databases
-  the wildcard will ignore the `postgres` database, template databases,
-  and those databases not allowing connections
+  temporarily contain the dump result on the assigned node as well as the
+  restored data and indexes. Once the import operation is complete, the
+  operator deletes the folder.
+- At least one database must be specified in the `initdb.import.databases` array.
+- Any role that's required by the imported databases must be specified in
+  `initdb.import.roles`, with these limitations:
+    - The following roles, if present, aren't imported:
+      postgres, streaming_replica, cnp_pooler_pgbouncer.
+    - The `SUPERUSER` option is removed from any imported role.
+- Wildcard `"*"` can be used as the only element in the `databases` or
+  `roles` arrays to import every object of the kind. When matching databases,
+  the wildcard ignores the `postgres` database, template databases,
+  and those databases not allowing connections.
 - After the clone procedure is done, `ANALYZE VERBOSE` is executed for every
```

Comment:
This is the first time I'm seeing the word "clone." Should this be "import"? Even if they both mean the same thing, we should stick with the same vocabulary, as it gets difficult for non-native English speakers to know whether we're talking about the same thing when we use two different words for the same thing. Or is cloning a separate part of the import process?